mozilla-release/mobile/android/base/java/org/mozilla/gecko/widget/SelfHidingLinearLayout.java

```diff
@@ -64,13 +66,15 @@ protected void onLayout(boolean changed, int l, int t, int r, int b) {
                 setVisibility(INVISIBLE);
             } else {
                 setVisibility(VISIBLE);
+                Telemetry.sendHomeSettingsShowTelemetry();
```

Comment:
`SelfHidingLinearLayout` is a generic class and it should not host any business logic. Please move it back to Activity Stream Panel